src/app/Instance/CreateInstance/CreateInstance.tsx

```diff
@@ -309,6 +391,37 @@ const CreateInstance = (props: CreateInstanceProps): JSX.Element => {
           isDisabled={formIsDisabled}
         />
 
+        <FormSection title={t("common.errorHandling")} titleElement="h2">
+          <FormGroup
+            label={t("common.errorHandlingMethod")}
+            fieldId={"error-handling-method"}
+            isRequired
+          >
+            <ErrorHandlingSelection
+              defaultMethod={ERROR_HANDLING_METHODS.default.value}
+              errorHandlingMethods={ERROR_HANDLING_METHODS}
+              isDisabled={formIsDisabled}
+              onMethodSelection={(errorMethod: string): void => {
+                setErrorHandlingParameters({});
+                if (errorMethod === ERROR_HANDLING_METHODS.default.value) {
+                  setErrorHandlingSchemaId(null);
+                } else {
+                  setErrorHandlingSchemaId(errorMethod);
+                }
+              }}
+            />
+          </FormGroup>
+          {errorHandlingSchema && !errorHandlingSchemaLoading && (
```

Comment:
This is causing the form to flicker when selecting between types.

IDK whether it's better to just check if the `setErrorHandlingSchemaId != null`?

Reply:
Hi @manstis, I tried to increase the delay for the schema API response, and it seems that the "flicking" effect is because when changing the method, the section disappears. As soon as we get the API response, the section appears again (with the proper content).
![changing_method_delayed](https://user-images.githubusercontent.com/22591802/186379098-3690eed5-8e16-497d-b1da-22756cb8d569.gif)

We have the same condition as in the processor edit page. I am unsure if we should handle it differently here since we have a modal with a dynamic height length.

Reply:
Fair enough. It is cosmetic and, if we have the same behaviour elsewhere, I will ignore (I have approved this PR!!)

When the selection changes we end up setting `errorHandlingSchemaLoading` to `true` meaning the form section is not rendered until the new schema retrieved and parsed. If we only checked `setErrorHandlingSchemaId != null` we'd show the existing schema form while loading the new one meaning no flicker. You could add `errorHandlingSchema && !errorHandlingSchemaLoading` to the `isDisabled` check resulting in the existing schema form being disabled while the new one loads to prevent naughty users from doing anything. Just a thought. No need to follow-up. PR approved "as is".